lib/fastly_nsq/message_queue/strategy.rb

```diff
@@ -1,12 +1,34 @@
-class Strategy
-  def self.for_queue
-    if MessageQueue::FALSY_VALUES.include?(ENV['FAKE_QUEUE'])
-      Nsq
-    elsif MessageQueue::TRUTHY_VALUES.include?(ENV['FAKE_QUEUE'])
-      FakeMessageQueue
-    else
-      message = "You must set ENV['FAKE_QUEUE'] to either true or false"
-      raise InvalidParameterError, message
-    end
+module MessageQueue::Strategy
+  module_function
```

Comment:
Same here: does this take a symbol as an argument?

http://ruby-doc.org/core-2.3.1/Module.html#method-i-module_function